transmart-copy/src/main/groovy/org/transmartproject/copy/Database.groovy

```diff
@@ -156,7 +157,7 @@ class Database implements AutoCloseable {
 
     final Map<Table, SimpleJdbcInsert> inserters = [:]
 
-    SimpleJdbcInsert getInserter(Table table, LinkedHashMap<String, Class> columns) {
+    SimpleJdbcInsert getInserter(Table table, Map<String, Class> columns) {
```

Comment:
I explicitly specified LinkedHashMap here, to make sure the order is preserved.

Reply:
Makes sense, indeed.